imagine/pipelines/pipeline.py

```diff
@@ -66,18 +71,30 @@ class Pipeline(BaseClass, metaclass=abc.ABCMeta):
         Likelihood object
     ensemble_size : int
         Number of observable realizations to be generated by the simulator
+    run_directory : str
+        Directory where the pipeline state and reports are saved.
     chains_directory : str
-        Path of the directory where the chains should be saved
+        Path of the directory where the chains should be saved. By default,
+        this is saved to a 'chains' subdirectory of `run_directory`.
     prior_correlations : dict
         Dictionary used to set up prior distribution correlations. If two
         parameters are A and B are correlated a priori, an entry should be
         added to the prior_correlations dictionary in the form
         `(name_A, name_B): True`, to extract the correlation from the samples
         (in the case of CustomPriors) or `(name_A, name_B): value` otherwise.
```

Comment:
I think we should explain more what we mean with 'value' here (i.e the nonlinear correlation coefficients). 
I will have a look the next days.